packages/cascara/src/modules/DataFile/DataFile.doc.mdx

```diff
@@ -20,7 +21,7 @@ Feel free to play around with the following code:
     lastName: 'Doe',
   }}
 >
-  <DataField
+  <DataFile
     attribute='firstName'
```

Comment:
@brianespinosa. This is another reason Snapshots changed, we corrected this typo in the component name.